provider/oci/testing/mocks.go

```diff
@@ -0,0 +1,622 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: common/interfaces.go
+
+// Package testing is a generated GoMock package.
```

Comment:
We have been adding _go generate_ directives at the sites of the interfaces that are being mocked.

Example: https://github.com/juju/juju/blob/9a70f7768a6dc67a95168f846272f1cf044254d0/service/systemd/service.go#L103

Makes it easy to update mocks for a changed interface.

Reply:
agreed. That way developers can just update the interface and don't have to know the magic invocation to get it regenerated to the right testing files, and can even just do:
go generate && go test

Reply:
done